freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -459,5 +483,19 @@ def _find_images_to_rebuild(self, errata_id, skip_nvrs=None):
             release_categories=release_categories,
             leaf_container_images=leaf_container_images,
             skip_nvrs=skip_nvrs,
+            restrict_to_ext_repos=restrict_to_ext_repos,
         )
         return batches
+
+    def _lookup_external_repos(self) -> list[str]:
+        """Fetches the external repositories to be used for lower criticality CVEs
+
+        :return: Names of the external repositories
+        :rtype: list of str
+        """
+        
+        url = conf.compliance_priority_repositories_remote_file
+        parsed_content = load_remote_yaml(url)
+        ext_repos = parsed_content["repositories"]
```

Comment:
`ext_repos` is redundant here. You can just return
`return parsed_content["repositories"]`

Even shorter:
`return load_remote_yaml(url).get("repositories")`
and you can delete `parsed_content` too.

Reply:
That's much better! Thanks a lot 😁